lib/seeing_is_believing/binary/config.rb

```diff
@@ -41,6 +41,7 @@ def to_s
       attribute(:help_screen)           { |c| Binary.help_screen c.markers }
       attribute(:lib_options)           { SeeingIsBelieving::Options.new }
       attribute(:annotator_options)     { AnnotatorOptions.new }
+      attribute(:port)                  { 5158 }
```

Comment:
It's leet for SisB 😉 . And I checked the big list of standard port numbers and didn't see anything listed for it.